### PR TITLE
Changed to smbus2 and added requirements.txt

### PR DESCRIPTION
**Pull request by mavaa**
I couldn't find the original smbus package, but I found this one, which seemed to work fine: https://pypi.org/project/smbus2/

They have a link to the original one, but it seems to be down.

Also added is a requirements.txt to more easily get the required packages. (Can be installed by `pip install -r requirements.txt`)

I also tested the script on a raspberry pi 3B+ which worked great, so if you want, I can also change the [Compatability](https://github.com/DFRobot/DFRobot_URM13/tree/master/python/raspberrypi#compatibility) here to "Work well" for Pi 3.

### CODE REVIEW

**Comment by qsjhyy** [comment]
Thank you very much. I will test and verify this in the future, and may add some modifications compatible with the old version of smbus.

**Comment by qsjhyy** [comment]
Hello, I just flashed a new Raspberry Pi image and didn't manually install the 'smbus' package. I imported it directly and it works fine. Maybe you can provide us with more information to understand the issue you encountered earlier.
image
image

**Comment by mavaa** [comment]
Oh, I didn't know it was a package you could install, I looked for it at pypi.org. It's probably missing for me because I used the minimal Raspberry Pi OS image. But curiously, the smbus2 apparently came pre-installed:

smbus

The reason I added it to requirements.txt was because I prefer to create a virtual-environment (venv) to hold all the packages I need. I also had to install this manually when I tested it on my own computer (where it also worked perfectly by the way 🙂 )

In summary, I'm not sure which version is best to use, so maybe this should just stay the way it is in your repo?

**Comment by qsjhyy** [comment]
It's worth mentioning that SMBus seems to be directly referenced in SMBus2, and it is indeed a pre-installed library. For most people, they need an easier way to use it, so my library will remain the same. However, when you use a more personalized approach, you will learn more, which is also quite good.
